workspace/workflows/config.py

```diff
@@ -130,7 +130,6 @@
     ],
     "opensafely-core/sqlrunner": [
         108481072,  # Dependabot updates (currently broken and expected to be replaced)
-        37329087,  # Auto merge Dependabot PRs
```

Comment:
This one is https://github.com/opensafely-core/sqlrunner/actions/workflows/auto-merge-dependabot-prs.yml which I think is different from the Dependabot updates. It's here because it does not run on the main branch